netbout-web/src/main/java/com/netbout/rest/TkStart.java

```diff
@@ -58,6 +58,7 @@
  *  this task citing a NullPointerException (See qulice issue #608 raised for
  *  the same) and then remove TkStart.java from qulice exceptions.
  */
+@SuppressWarnings("PMD.ConstructorShouldDoInitialization")
```

Comment:
@mkordas The violation is from qulice 0.16.2 and is related to the transient field **Date checked**, which is updated **act()** method. No sure it is possible to be only initialized in constructor as it has a specific logic during the **act()** method.